### DISCUSSION TITLE
"force_passive_ip" behind NAT

### DISCUSSION THREAD

**Discussion by D10DE** [drakkan/sftpgo#1211] [Q&A] [answered]
Hello and thank you a lot for the great server! 
There's a question:
Is there a way to set a domain name (i.e. the one I obtain from ddns) in the config? "force_passive_ip" accepts only IP, but my IP is dynamic.

**Comment by drakkan** [comment]
Hello,

this is not currently supported. I might consider adding something like this

http://www.proftpd.org/docs/contrib/mod_dynmasq.html

**Comment by drakkan** [comment, marked as answer]
https://github.com/drakkan/sftpgo/commit/a23fdea9e35d9f31babb0357e9507423961875e5

**Reply by D10DE** [reply]
Thank you! But I am sorry for a probably stupid question, but what exactly I am suppose to do with it? I understand that it must be added into the config file, but which version supports it? I see that the last version v2.4.4 has been released 3 weeks ago.
Anyways, I'm using portable v2.4.0 on windows, is there a way to upgrade without loosing all the setup of the virtual folders, users and etc (besides the config file)?

**Reply by drakkan** [reply]
> Thank you! But I am sorry for a probably stupid question, but what exactly I am suppose to do with it? I understand that it must be added into the config file, but which version supports it? I see that the last version v2.4.4 has been released 3 weeks ago. Anyways, I'm using portable v2.4.0 on windows, is there a way to upgrade without loosing all the setup of the virtual folders, users and etc (besides the config file)?

This is supported in the development version and will be included in v2.5.0. You can download build artifact from the actions page. To keep your current configurations check the upgrading docs
